.github/workflows/publish-docker-explorer.yml

```diff
@@ -1,40 +1,40 @@
-name: Build and publish docker image
+name: Build and publish explorer docker image
 
 on:
   push:
-    tags: [ "v[0-9]+.[0-9]+.[0-9]+*" ]
+    tags: ['alephium-explorer@[0-9]+.[0-9]+.[0-9]+*']
```

Comment:
This is the format that `bunx changeset tag` creates: `[package-name]@X.X.X`. I think it makes sense.